packages/config/__tests__/ProfileInfo.TeamConfig.test.ts

```diff
@@ -1112,6 +1112,38 @@ describe("TeamConfig ProfileInfo tests", () => {
         });
     });
 
+    describe("removeKnownProperty TeamConfig tests", () => {
+        it("should remove the property", async () => {
+            const profInfo = createNewProfInfo(teamProjDir);
+            await profInfo.readProfilesFromDisk();
+            jest.spyOn(profInfo as any, "getAllProfiles").mockReturnValue([{ profName: "test", profType: 'zosmf' }]);
+            jest.spyOn(profInfo as any, "mergeArgsForProfile").mockReturnValue({
+                knownArgs: [
+                    {
+                        argName: 'someProperty',
+                        dataType: 'string',
+                        argValue: 'someValue',
+                        argLoc: {
+                            locType: ProfLocType.TEAM_CONFIG
+                        }
+                    } as IProfArgAttrs
+                ]
+            });
+
+            const mergedArgs = profInfo.mergeArgsForProfile((profInfo.getAllProfiles('zosmf').find(v => v.profName === 'test')) as IProfAttrs);
+            const removeKnownPropertySpy = jest.spyOn(profInfo as any, 'removeKnownProperty').mockResolvedValue(true);
+            let caughtError;
+            try {
+                await profInfo.removeKnownProperty({mergedArgs: mergedArgs, property: 'someProperty'});
+            } catch (error) {
+                caughtError = error;
+            }
+
+            expect(caughtError).toBeUndefined();
+            expect(removeKnownPropertySpy).toHaveBeenCalledWith({mergedArgs, property: 'someProperty'});
+        });
```

Comment:
Please add some `expect` statements that verify the property has been removed from `knownArgs`, similar to the other test:
```
expect(afterUpdate.knownArgs.find(v => v.argName === "dummy")?.argValue).toEqual("example.com");
expect(afterRemove.knownArgs.find(v => v.argName === "dummy")).toBeUndefined();
```

Reply:
I changed the unit tests to include your feedback.